src/middlewared/middlewared/etc_files/rc.conf.py

```diff
@@ -287,6 +289,16 @@ def nut_config(middleware, context):
     yield 'nut_upsmon_enable="YES"'
 
 
+def openvpn_server_config(middleware, context):
+    yield 'openvpn_server_configfile="/etc/local/openvpn/server/openvpn_server.conf"'
```

Comment:
/etc/local is a freenas thing, we should probably use /usr/local/etc/ to make it clear the real location it means.